src/components/footer.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component } from '@angular/core';
+
+import AngularBolivia from '@assets/images/angular-bolivia.webp';
+import AngularWhite from '@assets/images/angular-white-logo.webp';
+import XLogo from '@assets/images/x-logo.svg';
+import LinkedinLogo from '@assets/images/linkedin-logo.svg';
+import GithubLogo from '@assets/images/github-logo.svg';
+import EmailLogo from '@assets/images/email-logo.svg';
+
+@Component({
+  selector: 'app-footer',
+  standalone: true,
+  template: `
+    <footer class="bg-[#111828] text-white py-12">
+      <div class="container mx-auto px-4">
+        <div class="flex flex-col md:flex-row md:justify-between gap-12">
+          <div class="flex flex-col md:items-start gap-4">
+            <div class="flex flex-row items-center gap-4">
+              <img [src]="ANGULAR_WHITE" alt="Logo de Angular" />
```

Comment:
Probably just to keep consistency, we'll have to save all texts in properties, otherwise, what's the pattern we follow in order to know what text goes inside a property and what not.